cluster/manager.go

```diff
@@ -1799,3 +1806,28 @@ func (c *ClusterManager) putNodeCacheEntry(nodeId string, node api.Node) {
 	defer c.nodeCacheLock.Unlock()
 	c.nodeCache[nodeId] = node
 }
+
+// ClusterManager implements osdconfig.ClusterConfig
+func (c *ClusterManager) GetClusterConf() (*osdconfig.ClusterConfig, error) {
+	return c.configManager.GetClusterConf()
+}
+
+func (c *ClusterManager) GetNodeConf(nodeID string) (*osdconfig.NodeConfig, error) {
+	return c.configManager.GetNodeConf(nodeID)
+}
+
+func (c *ClusterManager) SetClusterConf(config *osdconfig.ClusterConfig) error {
+	return c.configManager.SetClusterConf(config)
+}
+
+func (c *ClusterManager) SetNodeConf(config *osdconfig.NodeConfig) error {
+	return c.configManager.SetNodeConf(config)
+}
+
+func (c *ClusterManager) WatchCluster(name string, cb func(config *osdconfig.ClusterConfig) error) error {
```

Comment:
Create appropriate interfaces so that this object does not need to satisfy functions which will not be used.

Reply:
done